frontend/src/pages/TeacherProfile.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <Page blueBg>
+    <div class="teacher-profile-container">
+      <div class="teacher-profile-left">
+        <StudentProfileCard
```

Comment:
Let's rename reused components to be more generic. See #87

Reply:
Noted, we can do the renaming to another PR